frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js

```diff
@@ -115,7 +115,7 @@ const Review = ({
         ) : <div className="margin-bottom-3" />}
         <ApproverStatusList approverStatus={approverStatusList} />
         {hasIncompletePages && <IncompletePages incompletePages={incompletePages} />}
-        <Button disabled={hasIncompletePages} type="submit">{hasBeenReviewed ? 'Re-submit' : 'Submit'}</Button>
+        <Button disabled={hasIncompletePages} type="submit">{hasBeenReviewed ? 'Update report' : 'Submit'}</Button>
```

Comment:
Do you think going from Re-submit to 'update' will confuse users into thinking the report wont be submitted.

Reply:
what could go wrong ....